inc/Alien/Proj4/Makefile.PL

```diff
@@ -1,99 +0,0 @@
-# Proj4 detection process (need to rework using Devel::CheckLib)
-# 
-# (0) check WITH_PROJ
```

Comment:
No, because look in the actual Proj4.pm - it already is. The WITH_PROJ does NOT belong in there, but within the PDL Makefile.PL, where it already is.